tcms/templates/navbar.html

```diff
@@ -18,6 +18,13 @@
         <div class="collapse navbar-collapse navbar-collapse-1">
             <ul class="nav navbar-nav navbar-utility">
 
+                <li style="float:left">
+                    <a>
+                        <span id="tenant">
+                            {% block tenant %}{{ tenant_name }}{% endblock %}
+                        </span>
+                    </a>
+                </li>
```

Comment:
All of this should actually be a single empty block which will allow plugins to extend the contents. 

It will be up to the plugins to actually figure out what they need to do. 

See `block custom_login` in `registration/login.html`.